instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-resteasy-3.1/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/jaxrs/v2_0/ResteasyJettyHttpServerTest.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.jaxrs.v2_0;
+
+import io.opentelemetry.instrumentation.jaxrs.v2_0.JaxRsJettyHttpServerTest;
+import io.opentelemetry.instrumentation.testing.junit.InstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.http.HttpServerInstrumentationExtension;
+import io.opentelemetry.instrumentation.testing.junit.http.HttpServerTestOptions;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+class ResteasyJettyHttpServerTest extends JaxRsJettyHttpServerTest {
+  @RegisterExtension
+  static final InstrumentationExtension testing = HttpServerInstrumentationExtension.forAgent();
+
+  @Override
+  protected void configure(HttpServerTestOptions options) {
+    super.configure(options);
+
+    options.setResponseCodeOnNonStandardHttpMethod(500);
+  }
+
+  // resteasy 3.0.x does not support JAX-RS 2.1
+  @Override
+  protected boolean shouldTestCompletableStageAsync() {
+    return false;
+  }
```

Comment:
i dont see this config in the [groovy version of this class](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/f143d4763d8269b22370f34f5a9541a673e884ad/instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-resteasy-3.1/javaagent/src/test/groovy/ResteasyJettyHttpServerTest.groovy)